matrix-sdk-android/docs/packages.md

```diff
@@ -1,3 +1,7 @@
+# Package org.matrix.android.sdk._userstories
```

Comment:
The "_" should be removed since you renamed it without it.

Reply:
Good catch, thanks. Sadly Dokka does not complain :/